lib/fog/kubevirt/compute/models/pvcs.rb

```diff
@@ -1,10 +1,13 @@
 require 'fog/core/collection'
 require 'fog/kubevirt/compute/models/pvc'
+require 'fog/kubevirt/compute/utils/unit_converter'
```

Comment:
Is it needed if we already include the Utils module ?

Reply:
We only included module definition if `ExceptionWrapper` so we need it here as well.